client/src/sections/resources/UndergroundFireworksShop.tsx

```diff
@@ -0,0 +1,46 @@
+import Line from "../../components/Line";
+import Tile from "../../components/Tile";
+import useHave from "../../hooks/useHave";
+import { useProperties } from "../../hooks/useProperties";
+import { $effect } from "../../util/makeValue";
+
+const UndergroundFireworksShop = () => {
+  const { _fireworksShopEquipmentBought, _fireworksShopHatBought } =
+    useProperties({
+      _fireworksShopEquipmentBought: false,
+      _fireworksShopHatBought: false,
+    });
```

Comment:
This is great, although I just removed `useProperties` in favor of singular `useProperty`. :)